### PR TITLE
[launcher] Try to fix cloudbuild for launcher

### PR DESCRIPTION
**Pull request by jkl73**
Still experiencing empty files in OEM partition, try more logging and careful mount in the script.

Also moved `cloudbuild.yaml` from the root directory to launcher/ directory

### CODE REVIEW

File: launcher/image/cloudbuild.yaml [hunk 1]
```diff
@@ -54,7 +54,4 @@ steps:
 timeout: '3000s'
 
 options:
-  logging: CLOUD_LOGGING_ONLY
   dynamic_substitutions: true
-  pool:
-    name: 'projects/confidential-space-images-dev/locations/us-west1/workerPools/cs-image-build-vpc'
```
Comment (jkl73) [inline launcher/image/cloudbuild.yaml:60]: Remove this private pool requirement, so people can submit the image build in their own project

Comment (alexmwu) [reply launcher/image/cloudbuild.yaml:60]: Why did we have this?

File: launcher/image/fixup_oem.sh [hunk 2]
```diff
@@ -26,6 +26,19 @@ main() {
     sed -e 's|.*|[BEGIN_CS_CMDLINE]&[END_CS_CMDLINE]|g'
 
   umount /mnt/disks/efi
+
+  # Now the oem partition is sealed, we mount it to print it's content
+  if [[ ! -d /mnt/disks/oem ]]; then
+    mkdir /mnt/disks/oem
```
Comment (alexmwu) [inline launcher/image/fixup_oem.sh:32]: Cleanup this directory?

File: launcher/image/preload.sh [hunk 3]
```diff
@@ -132,8 +132,12 @@ main() {
   # Make sure cache is flushed for the OEM partition.
   sync ${OEM_PATH}
 
+  # Remount as read-only to avoid unexpected changes
```
Comment (alexmwu) [inline launcher/image/preload.sh:135]: Why do we need this? Are you worried that some other thing is modifying the OEM partition after the fact? Wouldn't that get reflected in the dm-verity?

Comment (jkl73) [reply launcher/image/preload.sh:135]: yes, the behavior I saw is the file is all good before the preload.sh finish, but somehow got corrupted before the dm-verity sealing. So dm-verity looks all good, but the files are still corrupted.

Remount it as read-only (as it should be before this script) just to make sure it didn't get written accidently

File: launcher/image/fixup_oem.sh (hunk 2 above)
Comment (jkl73) [reply launcher/image/fixup_oem.sh:32]: I think it is fine as this is not get persistent, just a tmp build environment

File: launcher/image/cloudbuild.yaml (hunk 1 above)
Comment (jkl73) [reply launcher/image/cloudbuild.yaml:60]: Used to have this to build images in the private pool. But I realize it's fine to have the image building step without private pool, just put tests in the private pool.

Good thing is people can now use this cloudbuild file directly to build image in their own project for testing and debugging

**Review by alexmwu — APPROVED** [review]

**Pull request merged**